src/Data/Attoparsec/Interpreter.hs

```diff
@@ -87,9 +88,24 @@ interpreterArgsParser progName = P.option "" sheBangLine *> interpreterComment
       *> ((end >> return "")
           <|> (P.space *> (P.manyTill anyCharNormalizeSpace end <?> "-}")))
 
+    horizontalSpace = P.satisfy P.isHorizontalSpace
+
     lineComment =  comment "--" (P.endOfLine <|> P.endOfInput)
+    literateLineComment = comment
+      (">" *> horizontalSpace *> "--")
+      (P.endOfLine <|> P.endOfInput)
     blockComment = comment "{-" (P.string "-}")
-    interpreterComment = lineComment <|> blockComment
+
+    literateBlockComment =
```

Comment:
I played around with fitting this into the `comment` machinery, but ended up with more `isLiterate` conditionals than I was comfortable with. I think I can rework the abstraction if this is important to you.